readme.md

```diff
@@ -26,19 +38,90 @@ isAbsoluteUrl('foo/bar');
 ```
 
 
-## Related
+## Options Parameters  
 
-See [is-relative-url](https://github.com/sindresorhus/is-relative-url) for the inverse.
+### httpOnly  
+
+Why use httpOnly?
+**Default:**    
+`isAbsoluteUrl('httpS://www.sindresorhus')` => `true`
+
+**With httpOnly**:  
+`isAbsoluteUrl('httpS://www.sindresorhus', {httpOnly: 'strictest')` => `false`  
+`isAbsoluteUrl('httpS://www.sindresorhus', {httpOnly: 'strict')` => `false`  
+`isAbsoluteUrl('httpS://www.sindresorhus', {httpOnly: 'loose')` => `true`
+
+This way it's possible to quickly differentiate a string that starts with the `http` protocol vs a "proper" absolut Web Url.
+
+#### > 'strictest'
+Checks for the `https://` (or non-secure `http`), `www` and the TLDomain (p.e. `.com`)
+
+```js
+isAbsoluteUrl('https://www.sindresorhus.com/path/to/file', {httpOnly: 'strictest');
```

Comment:
```suggestion
isAbsoluteUrl('https://www.sindresorhus.com/path/to/file', {httpOnly: 'strictest'});
```

Reply:
You are missing a `}` in all examples below as well.

Reply:
Ups... fixed it